scripts/command_helper.rb

```diff
@@ -146,7 +146,7 @@ def call(severity, time, program_name, message)
     def default_logger(verbose=false)
       if verbose
         logger = Logger.new(STDOUT)
-        logger.level = Logger::INFO
+        logger.level = Logger::DEBUG
```

Comment:
Ruby doesn't have a log.verbose though, so we can't make a distinction. for most programs I take debug and verbose to mean the same thing and there's usually only two levels of logging (normal and verbose/debug).  Some programs have 3 levels (normal, verbose, debug/extra verbose) but this one can't because of logger in ruby doesn't have a .verbose method